gpjax/kernels/base.py

```diff
@@ -32,8 +37,8 @@
 class AbstractKernel(Module):
     """Base kernel class."""
 
-    compute_engine: AbstractKernelComputation = static_field(DenseKernelComputation)
-    active_dims: List[int] = static_field(None)
+    compute_engine: Type[AbstractKernelComputation] = static_field(DenseKernelComputation)
```

Comment:
Why do we need `Type` here?

Reply:
Because it's a class, not an instance.  The type of `DenseKernelComputation` is `Type[DenseKernelComputation]`. Compare to `DenseKernelComputation()`, whose type is `DenseKernelComputation`.